Week3/More JavaScript.js

```diff
@@ -0,0 +1,180 @@
+};
+    function vehicleType(color,category) {
+        console.log("a " + color + " " + vehicleCategory[category] );        
+}
+     
+vehicleType("blue", 2);
+
+// 5. Can you write the following without the if statement, but with just as a single line with console.log(...);?
+
+    /* Yes, we can use ternary operator to achieve it. */
+
+console.log(3 === 3 ? "yes" : "no");
+    
+// 6. Create a function called vehicle, like before, but takes another parameter called age, 
+    //so that vehicle("blue", 1, 5) prints 'a blue used car'
+
+function vehicle(color, category, age) {
+    if (age >= 1) {
+        age = "used";
+    } else {
+        age = "new";
+    }
+    console.log("a " + color + " " + age + " " + vehicleCategory[category]);
+} 
+    
+vehicle("blue", 1, 5);
+
+// 7. Make a list of vehicles, you can add "motorbike", "caravan", "bike", or more.
+
+let listOfVehicle = ["motorbike", "caravan", "truck", "bike", "jeep"];
+
+// 8. How do you get the third element from that list?
+
+console.log(listOfVehicle[2]);
+
+// 9. Change the function vehicle to use the list of question 7. So that vehicle("green", 3, 1) prints "a green new bike".
+
+function newVehicle(color, category, age) {
+    if (age >= 4) {
+        age = "used";
+    } else {
+        age = "new";
+    }
+    console.log("a " + color + " " + age + " " + listOfVehicle[category]);
+}
+
+newVehicle("green", 3, 1);
+
+ 
+// 10. Use the list of vehicles to write an advertisement. So that it prints something like: "Amazing Joe's Garage, we service cars, motorbikes, caravans and bikes.". (Hint: use a for loop.)
+
+listOfVehicle = ["car", "motorbike", "caravan", "bike"];
+
+let nameOfVehicle = "";
+
+for (i = 1; i < listOfVehicle.length; i++) {
+  if (i < listOfVehicle.length - 1) {
+    nameOfVehicle += ", " + listOfVehicle[i] + "s";
+  } else {
+    nameOfVehicle += " and " + listOfVehicle[i] + "s.";
+  }
+}
+
+console.log("Amazing Joe's Garage, we service " + listOfVehicle[0] + "s" + nameOfVehicle);
+
+// 11. What if you add one more vehicle to the list, can you have that added to the advertisement without changing the code for question 10?
+
+listOfVehicle = ["car", "motorbike", "caravan", "bike"];
+
+listOfVehicle.push("jeep");
+
+nameOfVehicle = '';
+
+for (i = 1; i < listOfVehicle.length; i++) {
+  if (i < listOfVehicle.length - 1) {
+    nameOfVehicle += ", " + listOfVehicle[i] + "s";
+  } else {
+    nameOfVehicle += " and " + listOfVehicle[i] + "s.";
+  }
+}
+
+console.log("Amazing Joe's Garage, we service " + listOfVehicle[0] + "s" + nameOfVehicle);
+  
+
+// 12. Create an empty object.
+
+    const class23 = new Object ();
+
+// 13. Create an object that contains the teachers that you have had so far for the different modules.
+// 14. Add a property to the object you just created that contains the languages that they have taught you.
+
+    class23.teachers = 'Philipp, Rob, Unmesh & Bonan, Yash';
+
+    class23.languages = 'Html, CSS, Git, JavaScript1'
```

Comment:
Nicnice, could you use an `array` instead of a long `string`? Usually it's good practice to use an `array` for `multiple values`

Reply:
Thanks for the tip. I revised it with array.